changelog.d/20250417_162221_mlabeeb03_trigger_compose_started.md

```diff
@@ -0,0 +1 @@
+- [Improvement] Stop previous platform when a platform with a different name is executed with start, up, restart or run command. The stop command is only executed if platform to be stopped is in running state which can be checked with the docker compose ls command. (by @mlabeeb03)
```

Comment:
This changelog entry is not very informative. Actually, what we are changing is that the "dev" platform is now stopped on "local run" commands. Also, the COMPOSE_PROJECT_STARTED action is triggered just once per runtime.

Implementation details, such as `docker compose ls`, are uninteresting.